### DISCUSSION TITLE
Upgrade to Avalonia 11.0.0-preview5

### DISCUSSION THREAD

**Discussion by jp2masa** [SKProCH/Material.Icons#13] [General]
Is there any ongoing work to upgrade Avalonia to 11.0.0-preview5? If not, I would like to give it a shot.

I also noticed this rollback commit for preview4 -> preview1: https://github.com/SKProCH/Material.Icons/commit/25d6ac121c29ba39f1db2f4a1bc3b82ae4374b75, was there any specific problem?

**Comment by SKProCH** [comment]
Hello, @jp2masa.

No, i hasn't started upgrading to Avalonia 11.0.0-preview5 yet, you can start, if you want, sure.

About the rollback commit - I prefer that packages require the lowest possible version of dependencies. I that case - preview1 is the lowest possible.

**Reply by jp2masa** [reply]
>No, i hasn't started upgrading to Avalonia 11.0.0-preview5 yet, you can start, if you want, sure.

I'll work on it then.

> About the rollback commit - I prefer that packages require the lowest possible version of dependencies. I that case - preview1 is the lowest possible.

Makes sense. Unfortunately, preview5 has a lot of breaking changes, so upgrading is really necessary.

**Reply by SKProCH** [reply]
Yep. I've created the issue from this discussions, we can continue there.
